setup.py

```diff
@@ -54,7 +54,6 @@ def run(self):
     'Werkzeug==0.10.4',
 ]
 
-
```

Comment:
This newline is intentional (2 blank lines before a top-level class definition). No need to delete it.